frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -51,7 +51,7 @@ action.GitMachete.BaseOverrideForkPointAction.description-action-name=Fork point
 # {0} skipped intentionally (parent branch name is not needed)
 action.GitMachete.BaseOverrideForkPointAction.description=Override fork point of ''{1}''
 action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name.1=Select after which commit the range of commits belonging to
-action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name.2=branch starts
+action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name.2=branch starts:
```

Comment:
while we are here...
the suffixes `branch-name.1` and `branch-name.2` are relics. 
In the first place, they were actually containing branch names IIRC but now they do not make much sense

Reply:
![image](https://user-images.githubusercontent.com/19799111/96979382-6229ad00-151f-11eb-922b-9c128fc1e014.png)

Reply:
Ok, renamed